ba/js/lesson1/dist/script.dev.js

```diff
@@ -0,0 +1,57 @@
+"use strict";
+
+//Task 1
+var yourName = prompt("Your name..");
+console.log("Hello " + yourName + " !"); //Task 2
+
+var CURRENT_YEAR = 2020;
+var birthYear = prompt("When your were born?   ");
```

Comment:
use let instead of var
When were you born*

Reply:
Please do not  take to your consideration everything in file    /dist/script.dev.js
This file was automatically generated by CompileHero, not by me
My work is in script.js